### DISCUSSION TITLE
cannot find gps signal when using external Leo Bodnar GPSDO

### DISCUSSION THREAD

**Discussion by robertcluo** [srsran/srsRAN_Project#299] [General Help]
Hi,

I am trying to use Leo Bodnar GPSDO as the external clock reference. However, I connect the signal into Reference IN and set it as GPS. However, I cannot see the GPS signal. Is there any specific setting for getting the gpsdo work?

Thanks

**Comment by fllay** [comment]
I think you have to place the GPS antenna near window. It works for me.

**Comment by brendan-mcauliffe** [comment]
Hi @robertcluo, the Leo Bodnar requires a GPS lock, you will need to put the antenna next to a window, or somewhere with a clear view of the sky. It can take a couple of seconds for the lock to be acquired, the LED will stop blinking once it has the lock. 

If you continue having issues I suggest you consult the documentation or reach out to Leo Bodnar directly.

**Comment by pgawlowicz** [comment]
Hi,
"clock_source=gpsdo" is reserved for the internal USRP GPS receiver. With the LEO Bodnar please use "clock_source=external"
